fala/apps/adviser/tests/test_pagination_playwright.py

```diff
@@ -0,0 +1,19 @@
+from playwright.sync_api import expect
+from fala.playwright.setup import PlaywrightTestSetup
+
+
+class PaginationResults(PlaywrightTestSetup):
+
+    def test_results_pagination(self):
+        checkboxes = ["Crime"]
+
+        page = self.visit_results_page("M25 3JF", checkboxes)
+        expect(page.item_from_text("1576 results in order of closeness to M25 3JF.")).to_be_visible()
```

Comment:
I don't think we should hard code this number - I'd maybe omit that or possibly lose this line

Reply:
It has to be hardcoded as we want to check that the number of results does not change when the pages are changed, as a change would indicate that the categorisation was not being persisted when the pagination was used. I agree that this number could change in practice as providers leave or join, but I think that will not be frequently and the assurance provided by hardcoding outweighs the work required to update this if the number of results changes.